webapp/src/routes/home/game/ui/menuButton.js

```diff
@@ -0,0 +1,30 @@
+import { Container } from 'pixi.js';
+import { RectangleMenuBox } from './rectangleMenuBox';
+import { MenuIcon } from './menuIcon';
+
+export class MenuButton {
+  constructor({ x, y, width, height, type }) {
+    this._stage = new Container();
+
+    this.button = new RectangleMenuBox({
+      x: x,
+      y: y,
+      width: width,
+      height: height,
+    });
+
+    this.icon = new MenuIcon({
```

Comment:
Można było stworzyć odpowiedniego Sprite od razu w tym miejscu. Klasa `MenuIcon` wydaje się troche nadmiarowa

Reply:
zdecydowanie 👍 to jest kontener posiadający jedno dziecko w postaci sprite'a. Jako stage możnaby tego sprite'a od razu po prostu przypisać